bitnami/fluentd/templates/forwarder-configmap.yaml

```diff
@@ -77,6 +77,7 @@ data:
       @type kubernetes_metadata
     </filter>
 
+    {{- if .Values.aggregator.enabled -}}
```

Comment:
If the aggregator is disabled we should do something else with the logs. I think we can throw them to the stdout of each forwarder (see `aggregator-configmap.yaml`) and then add a note in the NOTES.txt about this new default configuration when aggregators are disabled.